specification/indices/rollover/IndicesRolloverRequest.ts

```diff
@@ -118,6 +118,12 @@ export interface Request extends RequestBase {
      * @server_default 1
      */
     wait_for_active_shards?: WaitForActiveShards
+    /**
+     * If set to true, the rollover action will only mark a data stream to signal that it needs to be rolled over at the next write.
+     * Only allowed on data streams.
+     * @server_default false
+     */
+    lazy?: boolean
```

Comment:
This parameter is read in https://github.com/elastic/elasticsearch/blob/d9e751602dc4a874ef381797844f80044889fbef/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestRolloverIndexAction.java#L58 since [8.13.0](https://github.com/elastic/elasticsearch/pull/103309).